packages/keycloak-user-management/src/components/UserGroupDetailView/index.tsx

```diff
@@ -26,8 +26,7 @@ export interface ViewDetailsProps {
  * @param props - detail view component props
  */
 const ViewDetails = (props: ViewDetailsProps) => {
-  const { groupId, keycloakBaseURL } = props;
-  const history = useHistory();
+  const { groupId, keycloakBaseURL, onClose } = props;
```

Comment:
IMO I think we should emulate how native web works over javascript where possible. for instance using url to preserve such information. also, in this case, retains intuitiveness of navigating back and forth in between the different views.

Reply:
This is not a new page/view it's just toggling the view details tab on and off. See the difference between the old behaviors [before here](https://fhir-web.opensrp-stage.smartregister.org/admin/users/groups) and [after here](https://web.opensrp-stage.smartregister.org/admin/users/groups)